sdk/src/main/java/io/matchmore/sdk/api/Call.kt

```diff
@@ -18,4 +18,20 @@ fun <T> Call<T>.async(success: SuccessCallback<T>?, error: ErrorCallback?) {
             error?.invoke(t)
         }
     })
+}
+
+fun Call<Void>.async(complete: CompleteCallback?, error: ErrorCallback?) {
```

Comment:
Because I don't want to have a Void/Unit callback that creates a Void object just to pass it as a result. So I create CompleteCallback. CompleteCallback doesn't have any result. Because it's different callback we need second extension for it.

Reply:
Ah, I see. other option would be to suppress the result. in swift you can do:
```
async { _ // suppresion happens here
}```

Reply:
Yes in kotlin you can do a suppress in the same way but I do it to simplify public API. 
Look at CRUD definition. Delete method use CompleteCallback instead od SuccessCallback:

`fun delete(item: T, complete: CompleteCallback? = null, error: ErrorCallback? = null)`

so it's a little bit cleaner to use API like this:

`deleteAll( { doSomething() } )` 

in compare to:

`deleteAll( { _ -> doSomething() } )` where suppressed param is Void/Unit.